dev-python/flask-paginate/flask-paginate-0.7.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( pypy3 python3_{6,7} )
+
+inherit distutils-r1
+
+MY_COMMIT="6445a67395912a5432014ec824ce4bf8fd13e22d"
+
+DESCRIPTION="Pagination support for flask"
+HOMEPAGE="https://flask-paginate.readthedocs.io"
+# https://github.com/lixxu/flask-paginate/issues/68
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
it should be `SRC_URI="https://github.com/lixxu/flask-paginate/archive/${PV}.tar.gz"` as we said in the previous PR.

Reply:
I tried it, the $S was no more working, also when I deleted it (and used the default value) put the PV in there. I mean it’s working.

Reply:
In dev-python/flask-themes it's also included for example.

Reply:
I didn't understood that.

I tried this and it seems it's working
```diff
 inherit distutils-r1

-MY_COMMIT="510ad833106134711868653fb597bf75ea8ac34f"
-
 DESCRIPTION="Pagination support for flask"
 HOMEPAGE="https://flask-paginate.readthedocs.io"
-# https://github.com/lixxu/flask-paginate/issues/68
-SRC_URI="https://github.com/lixxu/flask-paginate/archive/${MY_COMMIT}.tar.gz -> ${P}.tar.gz"
+SRC_URI="https://github.com/lixxu/flask-paginate/archive/${PV}.tar.gz"

 LICENSE="BSD"
 SLOT="0"
@@ -26,8 +23,6 @@ BDEPEND="
        test? ( dev-python/pytest[${PYTHON_USEDEP}] )
 "

-S="${WORKDIR}/${PN}-${MY_COMMIT}"
-
 python_test() {
        pytest -vv tests/tests.py || die "tests failed with ${EPYTHON}"
 }
```

btw there can be more than one maintainer. But I'm not specially interested on this pacakge so I don't have any problem to be removed from the maintainer list. did you ask in #gentoo-proxy-maint IRC channel? I think a dev have to open a ticket for you as proxy maintainer

Reply:
Sorry for my English here :) It doesn't work on my machine. I get a "The source directory '/var/tmp/portage/dev-python/flask-paginate-0.7.0/work/flask-paginate-0.7.0' doesn't exist". Should i just create this directory? I can't send messages in #gentoo-proxy-maint, it's a "can't send nick/channel". I tried several clients including Kiwi Web.

Reply:
Alright i forgot to remove the -> ${P}.tar.gz

Reply:
Now its working :)

Reply:
nice. I forgot that tarball should contain package name. 

You should fixup all commits and leave only one commit.

About the IRC channel, maybe you need to register your user.